### DISCUSSION TITLE
HashiCorp Vault Plugin System

### DISCUSSION THREAD

**Discussion by teneko** [rajanadar/VaultSharp#261] [Q&A] [answered]
Is there any way to implement a **backend** plugin over gRPC for HashiCorp and not HTTP as your library indicates?

I refer to this feature: https://www.vaultproject.io/docs/plugins

**Comment by hoerup** [comment, marked as answer]
If the question is regarding communication between vault process and any plugins, i think the question would be better suited over at https://github.com/hashicorp/vault
